lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -777,7 +777,17 @@ module Resources
             'start': '2012-05-17T15:33:18.000Z'
           }
         }
-      }
+      },
+      {
+        'path': '/name/0/id',
+        'op': 'test',
+        'value': 'CI-12467982-0'
```

Comment:
I dont think this is real id for name? it should look like 'CI-12724068-12724066-0' Thats the example in laurens read example, you should be able to use it

Reply:
https://github.com/cerner/fhir.cerner.com/commit/d6ef4e31556be3decb0db09531e1e82211b51d12#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR78

Reply:
Responded in the other comment.